.eslintignore

```diff
@@ -1,11 +1,6 @@
 node_modules
-/build
-/dist
```

Comment:
I think that we still need it. Same for build. Well, that's only useful for people how publish to npm/work on the build process.